docs/signals.rst

```diff
@@ -25,12 +25,6 @@ saving a historical record. Arguments passed to the signals include the followin
 For Many To Many signals you've got the following :
 
 .. glossary::
-    instance
-        The source model instance being saved
-
-    history_instance
-        The corresponding history record
-
```

Comment:
Were these intentionally removed?

Reply:
I was having doc compiling issues wen the same entry was listed in both glossary sections.  I think it was a re-base error?  I think it might be resolved now